docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation Instructions
+
+To prepare the local docs for preview:
+
+```bash
+$ python -m venv env
```

Comment:
- `python` doesn't exist in Ubuntu 22.04; `python3` should work everywhere.
- removing the `$` prompt would allow easier copy and paste into a shell

Reply:
Agreed. Updated.